vnext/Microsoft.ReactNative.IntegrationTests/ReactInstanceSettingsTests.cpp

```diff
@@ -42,6 +42,28 @@ TEST_CLASS (ReactInstanceSettingsTests) {
     });
     queueController.ShutdownQueueAsync().get();
   }
+
+  TEST_METHOD(JSDispatcher) {
+    auto queueController = DispatcherQueueController::CreateOnDedicatedThread();
+    auto uiDispatcher = queueController.DispatcherQueue();
+    TestCheck(uiDispatcher);
+
+    uiDispatcher.TryEnqueue([]() noexcept {
+      // use the UI thread dispatcher queue as an object to read/write JSDispatcherProperty
+      ReactInstanceSettings settings;
+      TestCheck(settings);
+      TestCheck(!settings.JSDispatcher());
+      TestCheck(!settings.Properties().Get(ReactDispatcherHelper::JSDispatcherProperty()));
+
+      auto jsDispatcher = settings.UIDispatcher();
```

Comment:
>auto jsDispatcher = settings.UIDispatcher(); [](start = 6, length = 44)

It would be better that the JSDispatcher is never the same as the UIDispatcher. It is very inefficient.

Reply:
Removed the test since ReactInstanceSettings.idl is not changed now